### ISSUE TITLE
Insufficient Contrast

### ISSUE THREAD

**Issue by Wilto** [Wilto/wilto-makes-food#23]
Good ol’ WCAG 2.0 SC 1.4.3.

Footer links (which are I think using global styles) and subnav links fail. Post/recipe `.intro` links fail at current size.
